drivers/hmis/spec/requests/hmis/ce/client_eligible_ce_opportunities_spec.rb

```diff
@@ -310,7 +310,7 @@
           opportunities = 30.times.map do
             build(:hmis_ce_opportunity, project: p1, data_source: ds1, candidate_pool: pool_veterans)
           end
-          Hmis::Ce::Opportunity.import!(opportunities)
+          opportunities.map(&:save!)
```

Comment:
Curious about the reason for this change -- I am guessing it's because `import!` skips validations?